elastic/kibana
# [Upgrade Assistant] Restrict UI to Kibana admins

Fixes https://github.com/elastic/kibana/issues/127563

This PR adds a check for the `manage` spaces capability before rendering the Upgrade Assistant UI. Per https://github.com/elastic/kibana/issues/127341#issuecomment-1063361313, this capability is equivalent to the "global admin" role in Kibana.

Since Kibana deprecations may exist across spaces, we need to ensure the user has access to manage all spaces, and restrict access to Upgrade Assistant if not.

### How to test
1. Create multiple spaces as the `elastic` user.
2. Create a user that has the `manage` cluster privilege, but only has access to the `default` space.
3. Log in as user created in step 2. Verify Upgrade Assistant displays with the prompt shown in the screenshot below

### Screenshots

<img width="712" alt="Screen Shot 2022-03-17 at 9 31 22 AM" src="https://user-images.githubusercontent.com/5226211/158820303-da14c750-ffbf-4090-adcf-9c1cd27f2ec2.png">
